WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Shipping Address Validation/ShippingLabelAddressFormViewModel.swift

```diff
@@ -286,8 +286,8 @@ extension ShippingLabelAddressFormViewModel {
 extension ShippingLabelAddressFormViewModel {
 
     enum AddressValidationError: Error {
-        case none
-        case remote(ShippingLabelAddressValidationError?)
+        case local
+        case remote(Error?)
```

Comment:
Small suggestion: this error can be non-optional right?

Reply:
Good catch! Fixed here: https://github.com/woocommerce/woocommerce-ios/pull/6735/commits/8e60d519197dfbaf3595d145caf1cf30b00f99ca